code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -596,6 +619,8 @@
 		return ..()
 
 /mob/living/simple_animal/relaymove(mob/user, direction)
+	if (stat == DEAD)
```

Comment:
youre telling me you could ride dead shit before this?

Reply:
I don't think this was ever actually used but theoretically yes